charts/prometheus/templates/prometheus-alerts-configmap.yaml

```diff
@@ -1029,7 +1029,7 @@ data:
             runbook_url:
             summary: Too many pods on '{{`{{ $labels.node }}`}}'
           expr: |
-            max(max(kubelet_running_pod_count{job="kubernetes-nodes"}) by(instance) * on(instance) group_left(node) kubelet_node_name{job="kubernetes-nodes"}) by(node) / max(kube_node_status_capacity_pods{job="{{.Release.Name}}-kube-state"}) by(node) > 0.95
+            sum by (node) ( kube_pod_info and on (pod) kube_pod_status_phase{phase="Running"} == 1 ) / on (node) kube_node_status_capacity_pods{job="{{.Release.Name}}-kube-state"} > 0.95
```

Comment:
Should this only include running? IE: what about pods that are initializing, crashlooping, etc.?

Reply:
Looks like [there are only 5 phases](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase):
- Pending
- Running
- Succeeded
- Failed
- Unknown

I guess "Completed" is "failed" or "succeeded". Should we include "Pending" in this list?

Reply:
after i did this, i found the following about fixing that query on the internets, which they basically have the same thing.  so not really sure about "not running" pods being in the count, thoughts?

https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/443/files

Reply:
one could ask even why 95% of pod capacity is "too many"

Reply:
this also correlates (from my testing) to be the same value as `k describe node <your node> | grep "Non-terminated Pods"`